src/blender_mcp/server.py

```
@@ -342,9 +356,19 @@ def execute_blender_code(ctx: Context, code: str) -> str:
+        # The addon prefixes these with "Code execution error: "; strip it so
+        # the user sees the underlying exception without two layers of framing.
+        msg = str(e)
+        prefix = "Code execution error: "
+        if msg.startswith(prefix):
+            msg = msg[len(prefix):]
+        logger.info(f"Blender Python error: {msg}")
+        return f"Blender Python error: {msg}"
     except Exception as e:
-        logger.error(f"Error executing code: {str(e)}")
-        return f"Error executing code: {str(e)}"
+        logger.error(f"Communication error executing code: {str(e)}")
```

Ruff (0.15.10)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)